### PR TITLE
Chocolatey Packages

### PR DESCRIPTION
**Pull request by AnthonyMastrean**
These packages work. Some notes...
## Package Icon

I speculated on the final package icon URL. I ran it through [cdn.rawgit.com](http://cdn.rawgit.com) site, per Chocolatey's "[creating packages](https://github.com/chocolatey/choco/wiki/CreatePackages#package-icon-guidelines)" page.

You can test it by pushing the icon to the master branch of your repository (or just accepting this PR), creating the package locally, and viewing it with the [NuGet Package Explorer](https://chocolatey.org/packages/NugetPackageExplorer). It will pull the package icon from the published location.

If it displays, then you're ready to publish.
## Package Version

You should see my notes in the README. Both the stable and playtest versions of this software are considered "prerelease" by NuGet (the technology behind Chocolatey's package format) and [SemVer](http://semver.org/). Given this, it doesn't make sense to have one package, where proper versions indicate stable and prerelease versions indicate playtests, until after you release 1.x.
## Uninstall

I noticed that the stable and playtests install to Programs and Features (in the Windows Control Panel) under the same name and overwrite one another. If a user installs both packages and uninstalls one, they may not be able to uninstall the other (Chocolatey won't find the uninstaller and may fail). A `choco uninstall foo -force` might work.

I don't have a recommendation, just noting for the record. You may decide not to publish the playtest version?

### CODE REVIEW

**Comment by AnthonyMastrean** [comment]
When you're ready to publish, I can transfer maintainership to your developers. I'll need one or more Chocolatey user names. And I'll remove myself from the list.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
-# OpenRA Chocolatey Package
-Automatically install and update using the [Chocolatey](https://github.com/chocolatey) binary package panager for Windows.
+# OpenRA Chocolatey Packages
+
+Install and update using the [Chocolatey](https://github.com/chocolatey) package manager for Windows. Because even the stable releases are considered "prerelease" due to the NuGet [version scheme](http://docs.nuget.org/Create/Versioning#prerelease-versions), we have to release two packages.
```
Comment (phrohdoh) [inline README.md:3]: > Because even the stable

Would be much better as:

> Even though the stable

File: openra/tools/uninstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  [CmdletBinding()]
+  param(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Name
+  )
+
+  $local_key     = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*'
+  $machine_key32 = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\*'
+  $machine_key64 = 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\*'
```
Comment (phrohdoh) [inline openra/tools/uninstall.ps1:37]: Is `Wow6432Node` a real tree node? OS X user here (so I'm not knowledgeable about this) and it sounds a bit bogus.

Comment (ferventcoder) [inline openra/tools/uninstall.ps1:37]: Yes it is.

Comment (AnthonyMastrean) [inline openra/tools/uninstall.ps1:37]: The Windows registry is messy :)

https://duckduckgo.com/?q=Wow6432Node

File: README.md (hunk 1 above)
Comment (AnthonyMastrean) [inline README.md:3]: I can update that now or you can tweak it if the PR is acceptable. The README isn't part of the Chocolatey packages, at any rate.

File: openra/tools/uninstall.ps1 (hunk 2 above)
Comment (ferventcoder) [inline openra/tools/uninstall.ps1:37]: Yeah... https://msdn.microsoft.com/en-gb/library/aa384129(VS.85).aspx (working on something related to this right now....)

**Comment by Mailaender** [comment]
> When you're ready to publish, I can transfer maintainership to your developers. I'll need one or more Chocolatey user names. And I'll remove myself from the list.

I just registered at chocolatey.org with the same username I use for GitHub. You don't need to remove yourself from the list. We can grant you maintainer rights for this repository, too.

**Comment by Mailaender** [comment]
@ScottNZ and @penev92 and @reaperrr and @cjshmyr: Are you as real Windows users also interested in maintaining this?

**Comment by cjshmyr** [comment]
I'm interested

**Comment by Mailaender** [comment]
Then you also need to https://chocolatey.org/account/Register and tell us your username.

**Comment by Mailaender** [comment]
> Given this, it doesn't make sense to have one package, where proper versions indicate stable and prerelease versions indicate playtests, until after you release 1.x.

We don't follow the semantic versioning scheme at all and there will never be `1.0` so maybe we can version this as `2015.03.29` instead of `0.0.0-pre20150329`?

**Comment by AnthonyMastrean** [comment]
@Mailaender Let's have the version conversation in another Issue, perhaps? I think we can come up with something :)

**Pull request merged**